tests/unit/admin/views/test_checks.py

```diff
@@ -0,0 +1,122 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+
+import pretend
+import pytest
+
+from pyramid.httpexceptions import HTTPNotFound
+
+from warehouse.admin.views import checks as views
+from warehouse.malware.models import MalwareCheckState
+
+from ....common.db.malware import MalwareCheckFactory
+
+
+class TestListChecks:
+    def test_get_checks_none(self, db_request):
```

Comment:
Nit: You could probably use `pytest.mark.parametrize` here to reduce these independent methods into a single `test_get_checks`, but the current structure also LGTM 🙂

Reply:
I looked a bit into parameterization, and I don't think it would work cleanly here because I need to call a `Factory.create()` method to generate the checks. It seems better suited for situations where the arguments are static objects (strings, lists, etc). Thanks for the tip though! This is also a super common testing pattern in golang.